bin/inference/pycbc_inference

```diff
@@ -102,6 +102,9 @@ parser.add_argument("--nprocesses", type=int, default=None,
     help="Number of processes to use. If not given then use maximum.")
 parser.add_argument("--skip-burn-in", action="store_true", default=False,
     help="Do not burn in with sampler.")
+parser.add_argument("--seed", type=int, default=0,
```

Comment:
If you want it to be 0, that's fine but I just want to make sure this is doing what's expected.